base/base_tests/small_set_test.cpp

```diff
@@ -0,0 +1,74 @@
+#include "testing/testing.hpp"
+
+#include "base/small_set.hpp"
+
+#include <algorithm>
+#include <iterator>
+#include <vector>
+
+using namespace base;
+
+namespace
+{
+UNIT_TEST(SmallSet_Empty)
+{
+  SmallSet<0> set;
```

Comment:
This is a test file, from my point of view it's not a problem, especially now, as we are started to use "std::" prefix.